controllers/loadtest_controller.go

```diff
@@ -56,64 +57,44 @@ type LoadTestReconciler struct {
 // or handling the termination of its pods.
 func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	var err error
-
 	log := r.Log.WithValues("loadtest", req.NamespacedName)
+
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
 	defer cancel()
 
-	// Fetch the current state of the world.
-
-	// Listing nodes will be required to achieve gang scheduling; however, this is
-	// not implemented at this time. Therefore, it is commented here for later:
-	//
-	//nodes := new(corev1.NodeList)
-	//if err = r.List(ctx, nodes); err != nil {
-	//	log.Error(err, "failed to list nodes")
-	//	return ctrl.Result{Requeue: true}, err
-	//}
-
-	pods := new(corev1.PodList)
-	if err = r.List(ctx, pods, client.InNamespace(req.Namespace)); err != nil {
-		log.Error(err, "failed to list pods", "namespace", req.Namespace)
-		return ctrl.Result{Requeue: true}, err
-	}
-
-	loadtests := new(grpcv1.LoadTestList)
-	if err = r.List(ctx, loadtests); err != nil {
-		log.Error(err, "failed to list loadtests")
-		return ctrl.Result{Requeue: true}, err
-	}
-
 	loadtest := new(grpcv1.LoadTest)
 	if err = r.Get(ctx, req.NamespacedName, loadtest); err != nil {
 		log.Error(err, "failed to get loadtest", "name", req.NamespacedName)
-
-		// do not requeue, the load test may have been deleted
```

Comment:
why remove the comments? I found them pretty useful :)

Reply:
Good to know. I did not realize they were useful. I thought they may be redundant since we had a comment every time we reconciled. I can see it can be useful in some contexts.